kogito-usertasks-custom-lifecycle-springboot/README.md

```diff
@@ -176,4 +200,8 @@ plus the approver who made the first one.
 		"nationality":"American"
 	}
 }
-```
\ No newline at end of file
+```
+
+You should see a similar message after performing the second line approval
+
+<p align="center"><img src="docs/images/secondLineApprovalCurl.png"></p>
```

Comment:
What does this screenshot means ?

Reply:
Hi @radtriste it is the log one sees after the curl command

Reply:
yep sorry, went to fast yesterday, thought it was a wrong screenshot.

For readability, can we have the logs as text instead of screenshot ?

Reply:
yep sorry, went to fast yesterday, thought it was a wrong screenshot.

For readability, can we have the logs as text instead of screenshot ?

Reply:
fixed @radtriste